submissions/Valkirin/pop-up/source.css

```diff
@@ -0,0 +1,179 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+*:focus {
+  outline: 2px solid #84f71a;
+}
+
+.header,
+.nav {
+  display: block;
+  height: auto;
+}
+
+body {
+  font: 16px Arial, serif;
+  color: #000;
+}
+
+.list {
+  list-style-type: none;
+}
+
+.header {
+  margin: 10px;
+}
+
+.nav {
+  position: relative;
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav_item {
+  padding-left: 20px;
+}
+
+.link {
+  text-decoration: none;
+  color: #000;
+}
+
+.link:hover,
+.button:hover,
+.ava:hover {
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+.popUpControl,
+.moreControl {
+  position: absolute;
+  overflow-y: auto;
+  clip: rect(0 0 0 0);
+}
+
+.button {
+  display: inline-block;
+  background-image: url("ico/favicon.ico");
+  background-size: contain;
+}
+
+.ava > img,
+.button {
+  height: 32px;
+  width: 32px;
+}
+
+.moreButton {
+  display: block;
+  margin-left: 42px;
+  background: #a2cbd8;
+  border-radius: 7px;
+  color: #a39b9b;
+  width: 160px;
+  text-align: center;
+  cursor: pointer;
+  line-height: 25px;
+  margin-top: 8px;
+}
+
+.moreButton:hover {
+  background: #45ccf9;
+  color: #000;
+}
+
+.box {
+  display: none;
+  position: absolute;
```

Comment:
please note: positioning will be broken if one of the `.box` parents has `position` property different from `static`. At the moment `.box` position calculated from `.nav`. You'll see the difference if add `position: relative` to appropriate `.nav_item` element (parent of the `.box`). Also there can be more popups for other items which should be positioned properly too. Also more `.nav_items` can be added after current item  which will broke arrow positioning. If you'll positioning `.box` in relation to it's parent (`.nav_item`) the current code will be more reusable.

Reply:
do you mean let nav_item position: relative and position .box to nav_item

Reply:
yep, exactly. `.nav_item` is logically a parent for appropriate `.box` alignment for this case.